### PR TITLE
Server docker running 

### PR DESCRIPTION
**Pull request by reginawang3495**
# Overview
Server docker runs with these changes, but _whether it runs properly is not verified_
image

# Change Overview
2 Main Changes Made, not 100% sure which one caused this, but 98% sure it's the first one.
## 1. package `bcrypt` -> `bcryptjs`
https://www.npmjs.com/package/bcryptjs
> Optimized bcrypt in JavaScript with zero dependencies. Compatible to the C++ bcrypt binding on node.js and also working in the browser.

**Possible TODO:** docker files installing `bcryptjs` are probably no longer necessary. E.G. [this line of rebuilding bcrypt here](https://github.com/uclaacm/membership-portal/blob/25b54fd4d4b6439191f617a24e433e0ee45f0fef/dev.Dockerfile#L12)

### 2. changing `python` installation to specify `python2`
Pretty sure this isn't the working change since we used python in only `bcrypt`, but `bcryptjs` doesn't use python
- this also includes setting the env for python `ENV PYTHONPATH /usr/lib/python2.7/site-packages`

# Node Env
Not necessarily relevant
```
node -v
v10.19.0
```
and 
```
nvm install v8.9.4
nvm use v8.9.4
```
# Other
Note: Won't be available until 9/19 so someone else should probably maintain this PR

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,15 +1,16 @@
 # load the alpine base image
 FROM alpine:3.13
 
-RUN apk add -U python make gcc g++ nodejs yarn npm
+RUN apk add -U python2 make gcc g++ nodejs yarn npm
+ENV PYTHONPATH /usr/lib/python2.7/site-packages
```
Comment (reginawang3495) [inline Dockerfile:5]: test this out -- it should work
```suggestion
RUN apk add -U make gcc g++ nodejs yarn npm
```

File: Dockerfile [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 
 # create the working directory
 RUN mkdir -p /var/www/membership
 
 # copy the package.json and shinkwrap file to app location
 # copy the node_modules to app location
 COPY package.json yarn.lock /var/www/membership/
-RUN cd /var/www/membership && yarn && npm rebuild bcrypt --build-from-source
+RUN cd /var/www/membership && yarn && npm rebuild bcryptjs --build-from-source
```
Comment (reginawang3495) [inline Dockerfile:13]: test this out -- rebuilding `bcryptjs` is probably not necessary 
```su/ggestion
RUN cd /var/www/membership && yarn
```

**Review by reginawang3495 — COMMENTED** [review]
test / apply these 2 comments throughout, it should be good to merge either way if it works on someone's computer

**Comment by ellieyhcheng** [comment]
Changed image version to not need to install various packages and run yarn install in the image itself, not on the host system. Together with the PR on the client and on the deployment repo, this should get the dev system up.

**Pull request merged**